v4/export/sql_type.go

```diff
@@ -18,6 +18,7 @@ var (
 )
 
 func initColTypeRowReceiverMap() {
+	colTypeRowReceiverMap = map[string]func() RowReceiverStringer{}
```

Comment:
Dup with line 11?

Reply:
I want to clear this map every time we init this.

Reply:
currently the only call of `initColTypeRowReceiverMap` is main routine 🤔 Or I guess that you could remove line 11